lib/gris/caching.rb

```diff
@@ -0,0 +1,21 @@
+module Gris
+  module Caching
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def cache_key_for(id)
```

Comment:
Should this be a private class method? Or would you expect that we'll be calling this directly in app code outside of `.cached_find` and `.expire_cache` methods?